container-search/src/main/java/com/yahoo/prelude/fastsearch/FS4FillInvoker.java

```diff
@@ -132,10 +125,11 @@ public void release() {
         }
     }
 
-    private int countFastHits(Result result) {
+    private int countFastHits(Result result, String summaryClass) {
```

Comment:
Should change name to countUnfilledFastHits

Reply:
Fixed